doc/source/runastrodriz.rst

```diff
@@ -61,3 +61,430 @@ for each input image.
 The '-n' option allows the user to specify the number of cores to be used in
 running AstroDrizzle.
 
+
+.. _runastrodriz-description:
+
+Description
+===========
+A lot of effort goes into trying to determine the WCS solution which provides
+closest agreement with the GAIA astrometry frame.  Combining the 
+images successfully requires the most accurate alignment possible between all 
+the input exposures taking into account the best available distortion model
+for the input exposures.  Being able to compare the combined image to other exposures
+taken of the same field at other times, or even with other telescopes, requires
+that the WCS be defined to come as close to the GAIA frame as possible.  The 
+processing done by `runastrodriz` attempts to not only apply the most current 
+distortion model, but also the best available pre-computed GAIA-based WCS 
+solutions while proceeding to determine it's own solution to the available GAIA
+reference stars for the field-of-view.  It then performs numerous checks to see 
+which of these solutions results in the most accurately aligned images to each 
+other, then to GAIA and selects that WCS solution.  This selected WCS solution 
+serves as the basis for creating the final, distorted-corrected, combined 
+drizzle products for the set of exposures being processed. 
+
+Overview
+--------
+The overall logic implemented by `runastrodriz` to generate the final set of 
+drizzle products involves creating multiple sets of drizzle products and ultimately
+selecting one as the 'best'.  The basic steps are outlined here, with following 
+sections providing additional detail on each step.
+
+    #. Initialization
```

Comment:
The numbering scheme was sort of confusing. There is only a single top-level item (Initialization). it left me expecting top level items and I thought I missed something

Reply:
Additional items are further down the document, after the section of sub-numbers for initialization.  This can be more clearly reviewed on the web version available online.